frontend/chat-plugin/src/components/chat/index.tsx

```diff
@@ -40,8 +40,16 @@ const Chat = (props: Props) => {
   const [isChatHidden, setIsChatHidden] = useState(true);
   const [messages, setMessages] = useState([welcomeMessage]);
 
+  const getResumeToken = () => {
+    const queryParams = new URLSearchParams(window.location.search);
+    if (queryParams.has('resume_token')) {
+      return queryParams.get('resume_token');
+    }
+    return null;
```

Comment:
`return queryParams.has('resume_token') ? queryParams.get('resume_token') : null` maybe the only very good use case for ternary operators :)

Reply:
also I just read the docs and `return queryParams.get('resume_token')` would return null if the key isn't there so it can be simplifed further (unless there's any other reason to check the key is there and I'm not aware of)